examples/service-catalog/OWNERS

```diff
@@ -3,7 +3,9 @@ reviewers:
   - pmorie
   - jboyd01
   - jpeeler
+  - adambkaplan
```

Comment:
Does DevEx own the service-catalog in origin?

Reply:
no